Apps/WebClient/src/ClientApp/src/components/DependentCardComponent.vue

```diff
@@ -898,7 +898,7 @@ export default class DependentCardComponent extends Vue {
                         borderless
                         :items="testRows"
                         class="w-100 mb-0"
-                        aria-describedby="COVID-19 Test Results"
+                        aria-describedby="COVID‑19 Test Results"
```

Comment:
Unrelated, but this aria tag is incorrect.  The [accessibility docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby) say:

> The aria-describedby attribute is very similar to [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) attribute. While the aria-labelledby lists the ids of the labels or elements that describe the essence of an object, the aria-describedby lists the ids of the descriptions or elements providing more information that the user might need. Both aria-labelledby and aria-describedby reference other elements to calculate a text alternative, but a label should be concise, while a description is intended to provide more verbose information; a label describes the essence of an object, while a description provides more information that the user might need.

So, since we have an element above that labels this table (the \<h4\>), we should be 1) using `aria-labelledby` and 2) providing it id of that element instead of text.  We'll need to update the \<h4\> to have an id.

Reply:
I've created a separate bug to track this issue: AB#14443.